src/Services/RawLocales.php

```diff
@@ -134,7 +134,7 @@ public function getDefault(): string
 
     public function getCurrent(): string
     {
-        return $this->getDefault();
+        return app()->getLocale();
```

Comment:
Right. For the native implementation, it would work like:

```php
    dump(config('app.locale')); // APP_LOCALE -> let's say "en"
    app()->setLocale('de');
    dd(config('app.locale')); // "de"
```

In the case of `$this->getDefault();` usage, the client would receive the value from the cache after the first call (mostly `APP_LOCALE`), even if the actual locale is changed. So for me, it works in a kind of "fallback" locale.

That's just thoughts here.